### ISSUE TITLE
Exclude procs process when using sudo

### ISSUE THREAD

**Issue by notDavid** [dalance/procs#606]
Hi there, fyi, not sure if it's an issue or some mis-understanding / configuration by me;

The procs process itself is only excluded when not using sudo:

```log
$ procs test123
 PID:▲ User │ TTY CPU MEM CPU Time │ Command
            │     [%] [%]          │

$ sudo procs test123
 PID:▲ User │ TTY  CPU MEM CPU Time │ Command
            │      [%] [%]          │
 4372  root │ s004 0.0 0.0 00:00:00 │ sudo procs test123
```

Best, David.

--
procs 0.14.4 (installed via HomeBrew)
macOS 14.3.1

**Comment by dalance** [comment]
This is expected behaviour.
The `4372` is `sudo` process, and the `sudo` process will launch `procs`.
`show_self = false` option hides the self `procs` process only, so `sudo` process will be shown.
If `show_self = false` are given, the result of `sudo procs` will be like below:

```
$ sudo procs test123
 PID:▲ User │ TTY  CPU MEM CPU Time │ Command
            │      [%] [%]          │
 4372  root │ s004 0.0 0.0 00:00:00 │ sudo procs test123
 4373  root │ s004 0.0 0.0 00:00:00 │ procs test123
```

**Comment by dalance** [comment]
I'll add `show_self_parents` option at #607.
It suppress to show the parents processes which have self as the only child.

For example, in the following process tree, "bash - sudo - procs" will be hidden.
The root "bash" will be shown because it has other process which are not related to procs as the children.

* bash
  * bash
    * sudo
      * procs
  * vim
  * ssh

How about it?

**Comment by dalance** [comment]
I've released v0.14.5.
